zk.js/yarn.lock

```diff
@@ -30,6 +30,35 @@
     superstruct "^0.15.4"
     toml "^3.0.0"
 
+"@coral-xyz/anchor@^0.27.0":
```

Comment:
this is weird I believe we are using anchor 0.28.0

Reply:
Might be an issue with rebasing!

Reply:
try to regenerate yarn.lock with  ```yarn install --force```